docs/UserGuide.adoc

```diff
@@ -127,6 +127,26 @@ Example: +
 `addfood -f pizza -c 30 -n all` +
 `addfood -f pasta -c 6 -n john -n bob -n daniel`
 
+==== Calculate cost `calculate`
+Calculate the total cost for each person or display cost owed for a particular person.
+
+Format: +
+`calculate [-n NAME]. . .` +
+
+Example: +
+`calculate` +
+`calculate -n john`
+`calculate -n john -alice`
+
+==== List items  `list food`
```

Comment:
Should we call it List Food instead?

Reply:
Yep agree, will do a PR after everyone else is done adding to make the changes unless Bob can change it while adding his part? @Jarrod-Bob

Reply:
Sure

Reply:
Yup, understood.